Bio/Seq.py

```diff
@@ -2643,6 +2680,16 @@ def join(self, other):
             return self.__class__(len(temp_data), character=self._character)
         return Seq(temp_data)
 
+    @property
+    def defined(self):
+        """Return True if the sequence is defined, False if undefined or partially defined.
+
+        Zero-length sequences are always considered to be defined.
+        """
+        if self._length == 0:
+            return True
+        return False
+
```

Comment:
Why not just ``return self._length == 0`` here?

Reply:
I find this easier to understand.